consumer/nf_management.go

```diff
@@ -123,3 +123,34 @@ func SendDeregisterNFInstance() (problemDetails *models.ProblemDetails, err erro
 	}
 	return
 }
+
+func SendUpdateNFInstance(patchItem []models.PatchItem) (nfProfile models.NfProfile, problemDetails *models.ProblemDetails, err error) {
+	logger.Consumerlog.Infof("Send Update NFInstance")
```

Comment:
Does not seem to be good log. Not adding any value. Move it to trace if otherwise or debug.